src/System.Data.SqlClient/tests/FunctionalTests/CloneTests.cs

```diff
@@ -30,7 +30,8 @@ public void CloneSqlCommand()
 
             SqlCommand command = connection.CreateCommand();
             command.CommandText = "select 1";
-            command.CommandTimeout = 45;
+            command.ResetCommandTimeout();
+            Assert.Equal(command.CommandTimeout, 30);
```

Comment:
The command timeout starts at 30, so it needs to be changed to something else to verify the behavior of ResetCommandTimeout. You can just keep the "command.CommandTimeout = 45;" line from before to achieve this.